post-processor/import/post-processor.go

```diff
@@ -146,6 +151,11 @@ func (p *PostProcessor) Configure(raws ...interface{}) error {
 			errs, fmt.Errorf("invalid boot mode '%s'. Only 'uefi' and 'legacy-bios' are allowed", p.config.BootMode))
 	}
 
+	if p.config.Platform != "linux" && p.config.Platform != "windows" {
```

Comment:
I would suggest accepting the empty string as well, but warn/error if the boot mode is uefi
```suggestion
	switch p.config.Platform {
	case "linux", "windows":
	case "":
		if p.config.BootMode == "uefi" {
			errs = packersdk.MultiErrorAppend(errs, "'platform' must be set for 'uefi' image imports")
		}
	default:
		errs = packersdk.MultiErrorAppend(errs, fmt.Errorf(
			"invalid platform '%s'. Only 'linux' and 'windows' are allowed",
			p.config.Platform))
```

Reply:
Updated to reflect suggestion. I also tested locally with platform set to windows and platform set to empty (when boot_mode is uefi)